packages/components/src/call-out/index.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+	Card,
+	__experimentalVStack as VStack,
+	__experimentalHStack as HStack,
+	__experimentalHeading as Heading,
+	Icon,
+} from '@wordpress/components';
+import { forwardRef } from 'react';
+import type { CallOutProps } from './types';
+import './styles.scss';
+
+function UnforwardedCallOut(
+	{
+		title,
+		icon,
+		illustrationSrc,
+		illustrationAlt,
+		description,
+		actions,
+		headingLevel = 6,
+		variant = 'default',
+	}: CallOutProps,
+	ref: React.ForwardedRef< HTMLElement >
+) {
+	return (
+		<Card ref={ ref } className={ `call-out call-out--${ variant }` }>
```

Comment:
Based on my understanding reading various DS PRs, I think we're supposed to prefix the classname with `a8c-components-`. See e.g. https://github.com/Automattic/wp-calypso/pull/102998#discussion_r2075443401